keyboards/handwired/daishi/daishi.h

```diff
@@ -0,0 +1,46 @@
+#ifndef DAISHI_H
+#define DAISHI_H
+
+#include "quantum.h"
+
+#define encoder_update(clockwise) encoder_update_user(uint8_t index, clockwise)
+
+// The first section contains all of the arguments
+// The second converts the arguments into a two-dimensional array
+#define KEYMAP( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K00A, K00B, K00C, K00D, K00E, K00F, K010, K011, \
+    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K10A, K10B, K10C, K10D, K10E, K10F, K110, K111, \
+    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K20A, K20B, K20C, K20D, K20E, K20F, K210, K211, \
+    K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K30A, K30B, K30C, K30D, K30E, K30F, K310, K311, \
+    K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K40A, K40B, K40C, K40D, K40E, K40F, K410, K411, \
+    K500, K501, K502, K503, K504, K505, K506, K507, K508, K509, K50A, K50B, K50C, K50D, K50E, K50F, K510, K511, \
+    K600, K601, K602, K603, K604, K605, K606, K607, K608, K609, K60A, K60B, K60C, K60D, K60E, K60F, K610, K611 \
+) { \
+    { K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K00A, K00B, K00C, K00D, K00E, K00F, K010, K011 }, \
+    { K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K10A, K10B, K10C, K10D, K10E, K10F, K110, K111 }, \
+    { K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K20A, K20B, K20C, K20D, K20E, K20F, K210, K211 }, \
+    { K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K30A, K30B, K30C, K30D, K30E, K30F, K310, K311 }, \
+    { K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K40A, K40B, K40C, K40D, K40E, K40F, K410, K411 }, \
+    { K500, K501, K502, K503, K504, K505, K506, K507, K508, K509, K50A, K50B, K50C, K50D, K50E, K50F, K510, K511 }, \
+    { K600, K601, K602, K603, K604, K605, K606, K607, K608, K609, K60A, K60B, K60C, K60D, K60E, K60F, K610, K611 }  \
+}
+
+#define KC_KEYMAP( \
```

Comment:
This layout macro should be renamed:

```suggestion
#define LAYOUT_kc( \
```